projects/Mallard/src/hooks/use-is-signed-in.ts

```diff
@@ -0,0 +1,21 @@
+import { useEffect, useState } from 'react'
+import { membershipAccessTokenKeychain } from 'src/authentication/keychain'
+
+/**
+ *
+ * Returns
+ *  null - means pending on the async request
+ *  false - no currently loged in member
+ *  true - is signed in
+ */
```

Comment:
Maybe this should return a discriminated union?

Reply:
Changed to return an enum